pytest_fixtures/component/user.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from nailgun import entities
+
+
+@pytest.fixture
+def user_not_exists(request):
+    """Remove a user if it exists. Return whether changes have been applied."""
+    users = entities.User().search(query={'search': f'login={request.param}'})
```

Comment:
If possible can we use a sat_object directly for searching the user?

Reply:
I can see why you want that. But how? This fixture could take fixture `parametrized_enrolled_sat`, but then it would only be useful in this context, wouldn't be universally usable in different tests that might want to use it.